src/main/webapp/app/components/tabs/CurationDataValidationTab.tsx

```diff
@@ -0,0 +1,330 @@
+import { faRedo } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { DANGER, GREY, SUCCESS } from 'app/config/colors';
+import Collapsible from 'app/pages/curation/collapsible/Collapsible';
+import OncoKBTable, { SearchColumn } from 'app/shared/table/OncoKBTable';
+import React, { DependencyList, useCallback, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { Col, Form, FormGroup, Input, Row, Label, Button } from 'reactstrap';
+import Tabs from './tabs';
+
+type ValidationResultFilter = {
+  success: boolean;
+  error: boolean;
+};
+
+type ValidationSearchFormProps = {
+  onChange: (validationResultFilter: ValidationResultFilter) => void;
+  className?: string;
+  disabled?: boolean;
+};
+
+function ValidationSearchForm({ onChange, className, disabled }: ValidationSearchFormProps) {
+  const [validationResultFilter, setValidationResultFilter] = React.useState<ValidationResultFilter>({
+    success: false,
+    error: true,
+  });
+  useEffect(() => {
+    onChange(validationResultFilter);
+  }, [validationResultFilter]);
+
+  return (
+    <Form className={className}>
+      <Row>
+        <Col>
```

Comment:
Could you remove the padding on the column so it lines up nicely with the tab titles?